cookbooks/tile/templates/default/export.erb

```diff
@@ -180,7 +181,7 @@ if 'HTTP_REFERER' not in os.environ:
 if '_osm_totp_token' in cookies:
   token = cookies['_osm_totp_token'].value
 elif os.environ['REQUEST_METHOD'] != 'OPTIONS':
-  token = form.getfirst("token")
+  token = form.get("token", [None])[0]
```

Comment:
We can presumably just make this a straight `else` now and get rid of the options check?